### PR TITLE
Add release schedule info to the about page

### LINKED ISSUE #1970: Automated release details
We should add something somewhere that lets users know what day and time they can look forward to an update to app.destinyitemmanager.com

Maybe in DIM, on the website, GitHub/README, Twitter?

Something along the lines of...
> DIM automatically updates with new features every Friday at X:00

### PR DESCRIPTION
**Pull request by bhollis**
Fixes #1970.

### CODE REVIEW

File: src/i18n/dim_en.json [hunk 1]
```diff
@@ -586,7 +586,9 @@
       "TranslationText": "We use {{link}} for ease of translation. If you want to improve one of DIM's translations, join the team.",
       "Twitter": "Twitter",
       "TwitterHelp": "Be sure to follow us to stay on top of updates and realtime application state information.",
-      "Version": "Version {{version}} ({{flavor}}), built on {{date}}"
+      "Version": "Version {{version}} ({{flavor}}), built on {{date}}",
+      "ReleaseSchedule": "This version of DIM is updated once a week, in the afternoon on Saturdays, US Pacific time.",
```
Comment (delphiactual) [inline src/i18n/dim_en.json]: Schedule_Release, and Schedule_Beta

File: src/views/about.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <div class="ngdialog-inner-content">
     <a ng-href="{{ $DIM_CHANGELOG }}" target="_blank" rel="noopener noreferrer"><span ng-i18next="[i18next]({ version: $DIM_VERSION, flavor: $DIM_FLAVOR, date: $DIM_BUILD_DATE })Views.About.Version"></span></a>
     <p ng-i18next="Views.About.HowItsMade"></p>
+    <p ng-if="$DIM_FLAVOR == 'release'" ng-i18next="Views.About.ReleaseSchedule"></p>
```
Comment (delphiactual) [inline src/views/about.html]: <p ng-i18next="[i18next]({ context: $DIM_FLAVOR})Views.About.Schedule"></p>

File: src/i18n/dim_en.json (hunk 1 above)
Comment (delphiactual) [inline src/i18n/dim_en.json]: Also move them to the alphabetical location or run `grunt download_translation` or register another task that just sorts dim_en.json for PRs towards dim_en.json.

Comment (bhollis) [inline src/i18n/dim_en.json]: Hmm, not sure how to do what you're talking about. Want to submit a PR (or push a change onto this branch)?

**Comment by delphiactual** [comment]
#1987

**Pull request closed without merging**